src/schema.graphql

```diff
@@ -45,12 +46,53 @@ type Mutation {
   deleteOrganization(id: ID!): Boolean!
 }
 
+type MultiLingualField {
+  en_CA: String
+  fr_CA: String
+  id: String
+}
+
+type OrgChart {
+  boxes: [OrgChartBox!]!
+  lines: [OrgChartLine!]!
+  miniboxes: [OrgChartBox!]!
+  minilines: [OrgChartLine!]!
+}
+
+type OrgChartBox {
+  id: String!
+  x: Float!
+  y: Float!
+  width: Float
+  height: Float
+  on_path: Boolean
+  node: OrgChartNode!
+}
+
+type OrgChartNode {
```

Comment:
The OrgChartNode type is declared but never used?

Reply:
It is used, directly above inside the OrgChartBox type.